RecoHI/HiTracking/python/HiTracking_cff.py

```diff
@@ -25,62 +25,69 @@
 from RecoJets.JetAssociationProducers.trackExtrapolator_cfi import *
 trackExtrapolator.trackSrc = cms.InputTag("hiGeneralTracks")
 
-hiTracking_noRegitMu = cms.Sequence(
-    hiBasicTracking
-    *hiDetachedTripletStep
-    *hiLowPtTripletStep
-    *hiPixelPairStep
+hiTracking_noRegitMuTask = cms.Task(
+    hiBasicTrackingTask
+    ,hiDetachedTripletStepTask
+    ,hiLowPtTripletStepTask
+    ,hiPixelPairStepTask
     )
+hiTracking_noRegitMu = cms.Sequence(hiTracking_noRegitMuTask)
```

Comment:
This was only needed here internally: sequence not needed any more now, please remove